LICENSE

```diff
@@ -0,0 +1,21 @@
+MIT License
+
+Copyright (c) 2019 MacPorts
```

Comment:
Should we use ", The MacPorts Project." here? It's used in macports-base and ports.

Reply:
I actually wanted to ask because I wasn't sure what to use as the copyright holder. You are probably right that `The MacPorts Project` might make more sense, in particular if used in other repos.